---
string capture_mode
string fringe_coding_mode
